### PR TITLE
theme designer: add custom theme to export

### PR DESCRIPTION
**Pull request by emmayjiang**
The export link now also exports customizations!
image

To-Do List: https://github.com/microsoft/fluentui/projects/30?fullscreen=true

### CODE REVIEW

**Comment by emmayjiang** [comment]
image

File: packages/react-components/theme-designer/src/ThemeDesigner.tsx [hunk 1]
```diff
@@ -17,17 +17,17 @@ export const ThemeDesigner: React.FC<ThemeDesignerProps> = props => {
 
   const [state, dispatchState] = useThemeDesignerReducer();
 
-  const overridenTheme = overrideTheme(state.theme, state.brand, state.overrides);
+  const themeOverride = themeOverrides(state.theme, state.brand, state.overrides);
```
Comment (micahgodbolt) [inline packages/react-components/theme-designer/src/ThemeDesigner.tsx:20]: seems this function no longer needs theme. also, there's no need for the overrides to have a hex value. They could just be palette['40'] or whatever, and pull directly from the palette

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/5e326ed2e7068e8eb75df4a54495c27206ec831b'>5e326ed2e7068e8eb75df4a54495c27206ec831b</a></sub>
<div id="bundleSizeComment9423"/>

**Review by micahgodbolt — APPROVED** [review]
works for now, a few improvements to be put on the backlog unrelated to this work

**Pull request merged**